### ISSUE TITLE
Looks good! but...

### ISSUE THREAD

**Issue by oferRounds** [aelesia/react-native-cross-actionsheet#2]
I think it’s better to have the Android use their Material design look, and not the one of iOS

https://material.io/components/sheets-bottom

**Comment by aelesia** [comment]
Hi, yup I did consider using Material design at first, but the problem is that I don't currently have a way to get users to "input" an icon from RN into Android. Without the icons on the left side, the menus just look boring and empty, and thus the decision to adopt the iOS style of center aligned menus instead.

I'm considering adding the ability to add icons in the future from [rn-vector-icons](https://github.com/oblador/react-native-vector-icons). When I do incorporate icons I'll allow the developers to configure between the Material-based theme or the iOS theme.

**Comment by oferRounds** [comment]
Hi @aelesia got you. In my opinion, the idea of React Native is not to enforce one platform UI/UX on another because the code is shared. So I think it is kind of a must. I agree that without the icons it looks odd, and I think supporting rn-vector-icons is a good idea. Regarding the custom icons – why it’s not possible just allow him to pass a file (i.e., 'require('pathToTheLocalAssest')?

**Comment by aelesia** [comment]
I envision it to be easy for users by allowing them to choose their icons from a predefined list, something like:

```
ActionSheet.options([
  {text:"Okay", icon:"icon-tick"},
  {text:"Delete", icon:"icon-trash"},
  {text:"Cancel", icon:"icon-cancel"}
])
```

Requiring a file path takes quite a bit more effort. They have to search for the file, download it, resize it, link the assets, and do the same for iOS. In between it's easy to make mistakes. I'd like to make this process as easy and straightforward for developers by only allowing predefined icons. :)

**Comment by oferRounds** [comment]
Yes, I agree it takes more effort, but IMO it’s required... BTW – no linking, and no adding to both platforms. Just add it once to the JS bundle – and that’s it. The only thing which is required is to deal with the size, but it’s not such a big deal IMO

**Comment by stevenkitter** [comment]
_ActionSheetAndroidModule.ActionSheetAndroidModule.options err
